### DISCUSSION TITLE
Multiple mailing lists in a form [version 5 migration]

### DISCUSSION THREAD

**Discussion by rellafella** [solspace/craft-freeform#1451] [Feature Requests]
### What happened?

Previously we could add multiple fields for the same integration that would allow users to opt into multiple lists from the same Email Marketing integration in a single form.
Since integrations are no longer fields, it seems that we can only have a single mailing list attached to each form for each integration.

### How can we reproduce this?

1. Buy a license for the Freeform plugin
2. Create a form and email marketing integration in Freeform 4
3. Add multiple email marketing integration fields to this form and attach them to different lists
4. Back up your integration metadata because this will be deleted in the Freeform 5 migrations
5. Write your own migrations to migrate your integrations from Freeform 4 to Freeform 5
6. Review your form and check that it has retained all of its previous functionality

### Environment

- Freeform Pro 5.x
- Craft 4.x

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

4.x

**Comment by kjmartens** [comment]
Hi @rellafella,

Unfortunately, this is currently a limitation in Freeform 5. In order to achieve the same effect, you will need to add multiple copies of that integration and configure each one to work with a different checkbox field, etc.

It's on our list for future consideration to build out the ability to include a select dropdown or checkbox group, etc where you can have multiple mailing lists mapped on a single integration. 🙂

**Comment by thisisjamessmith** [comment]
Just ran into this issue too whilst attempting the v5 upgrade. In 4 I was using multiple DotDigital checkboxes for signing up to 6 different lists. Would be great to hear if there's any progress on making this more client-friendly. Thanks for describing the workaround though.
